include/openthread/platform/radio.h

```diff
@@ -212,21 +212,14 @@ typedef otCryptoKeyRef otMacKeyRef;
  * This structure represents a MAC Key.
  *
  */
-OT_TOOL_PACKED_BEGIN
-struct otMacKeyMaterial
+typedef struct otMacKeyMaterial
 {
     union
     {
         otMacKeyRef mKeyRef; ///< Reference to the key stored.
         otMacKey    mKey;    ///< Key stored as literal.
     } mKeyMaterial;
-} OT_TOOL_PACKED_END;
-
-/**
- * This structure represents a MAC Key reference.
- *
- */
-typedef struct otMacKeyMaterial otMacKeyMaterial;
+} otMacKeyMaterial;
```

Comment:
@hemanth-silabs can you please help double-check that the removal of "packed is ok. I think it should be fine. - 

- From what I recall, in earlier versions of the PSA PR, we used the `KeyMaterial` directly in `KeyManager::HashKeys` definition (which was then used to update MAC/MLE keys) and therefore it needed to be packed. But in the code now we are using `Mac::Key` directly in `HashKey` and then we  populate the `KeyMaterial` instances using `KeyMaterial::SetFrom()` (which either copies the key or saves it as a `KeyRef` in storage).

Reply:
@hemanth-silabs , can you help provide your thoughts on the above?

Reply:
@abtink sorry for the delay in response. I did not see this notification due to some issue with my mail filter. You are right. it needed to be packed only for the `KeyManager::HashKeys`, now that `KeyManager::HashKeys` has been updated, this need not be packed.